packaging/windows/inc/crtvariables.wxi

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<Include xmlns="http://schemas.microsoft.com/wix/2006/wi">
+
+  <!-- These values were generated by running 'heat payload' against the CRTs downloaded from the fwlinks -->
+
+  <?define Crt_CertificatePublicKey="52868DFCA6E3AF2632389E6C1EE7D0468D3797D0"?>
+  <?define Crt_CertificateThumbprint="3BDA323E552DB1FDE5F4FBEE75D6D5B2B187EEDC"?>
+  <?define Crt_Description="Microsoft Visual C++ 2015 Redistributable ($(var.Platform)) - 14.0.23506"?>
+  <?define Crt_ProductName="Microsoft Visual C++ 2015 Redistributable ($(var.Platform)) - 14.0.23506"?>
+  <?define Crt_Version="14.0.23506.0"?>
+  <?define Crt_VersionRegistyKeyValue="v14.0.23506.00"?>
+
+  <?if $(var.Platform)=x86?>
+  <?define Crt_DownloadUrl="http://go.microsoft.com/fwlink/?LinkID=615459&amp;clcid=0x409"?>
```

Comment:
If you want to localize them, definitely. In general, I minimize what goes in to .wxi files and preprocessor in general since people tend to very quickly abuse the whole thing. If a value isn't shared, just put it where it goes.